datadog/resource_datadog_synthetics_test_.go

```diff
@@ -1773,6 +1794,19 @@ func buildSyntheticsBrowserTestStruct(d *schema.ResourceData) *datadogV1.Synthet
 			}
 			options.SetRestrictedRoles(roles)
 		}
+
+		if rum_settings, ok := d.GetOk("options_list.0.rum_settings.0"); ok {
+			settings := rum_settings.(map[string]interface{})
+			applicationId := settings["application_id"]
+			clientTokenId := settings["client_token_id"]
+
+			rumSettings := datadogV1.SyntheticsBrowserTestRumSettings{}
+			rumSettings.SetIsEnabled(true)
```

Comment:
I think it's better to add another property for this instead of always defaulting to true.

Reply:
I thought it would be easier to use instead of having  to explicitly have `is_enabled = true` that would seem a bit redundant. 
Basically either you have a configuration and it's enabled, or you don't. But I can add it if you think it can be useful in some case.

Reply:
Yeah i think its better to have it as a configurable property with a default of `true`